index.js

```diff
@@ -74,7 +74,7 @@ For each function call return a json object with function name and arguments wit
 `
 
 /*
-Use the following pydantic model json schema for each tool call you will make:
+Use the following pedantic model json schema for each tool call you will make:
```

Comment:
looks like Pydantic Model is actually a [thing](https://github.com/NousResearch/Hermes-Function-Calling?tab=readme-ov-file#adding-custom-pydantic-model). will roll back that change